vectordb_bench/backend/clients/aws_opensearch/config.py

```diff
@@ -0,0 +1,60 @@
+from enum import Enum
+from pydantic import SecretStr, BaseModel
+
+from ..api import DBConfig, DBCaseConfig, MetricType, IndexType
+
+
+class AWSOpenSearchConfig(DBConfig, BaseModel):
+    host: str = (
+        "xxxxxx.us-west-2.es.amazonaws.com"
+    )
+    port: int = 443
+    user: str = "admin"
+    password: SecretStr = "xxxxxx"
```

Comment:
these default values will be display and used in front page, please just leave it empty
![image](https://github.com/user-attachments/assets/b77ca587-384b-4e02-8b01-806d9045c9da)

Reply:
done